### DISCUSSION TITLE
Decrease or cancel mining rewards in the current epoch for identities that have produced flips that have been reported in the validation session

### DISCUSSION THREAD

**Discussion by ltraveler** [idena-network/idena-docs#131] [IIPs]
To increase overall flips quality and make massive flip production process harder I suggest to decrease mining rewards or even epochally cancel it in case if identity have produced flips that have been reported.
The proposal could be considered as future evolution of IIP-7 and would motivate people put more attention on the flips production process apart from creating extra flips also it could motivate people to do report procedure more precise.

**Comment by illusiontorment** [comment]
Hey, Traveler,

a question: What would motivate people to stop reporting good flips just because they have more reports than bad flips in the long session just for the fun of it or to mess with other people's rewards?

It is bad enough to lose validation rewards due to people reporting randomly...

**Reply by ltraveler** [reply]
> a question: What would motivate people to stop reporting good flips just because they have more reports than bad flips in the long session just for the fun of it or to mess with other people's rewards?
> 
> It is bad enough to lose validation rewards due to people reporting randomly...

IMHO, the weak component in that IIP are farms.
Regular people by knowing that your flips are also could be reported randomly never gonna click report button on good flip just because of a very small chance to be rewarded by coincidence - do as you would be done by.

**Reply by codeheroo** [reply]
farms have been already discriminated too much. because of the lasts IIPs there's no awards for newcomers. to get normal rewards, they need to buy around 10k coins. it's too much money moreover, that you can create flips with 3 photos can disappoint someone and then, this flip becomes reported and they lose money. it's unfair

**Comment by Noobmasser** [comment]
There's high risk of getting punished for nothing. I think if some measures are to be taken, then these have to be for consecutive reports to diminish the chance of random unfortunate circumstances.

**Comment by ltraveler** [comment]
**In telegram we have had pretty interesting discussion:**

>by Teajay666
> something like N-number of reported flips across N-number of validations = suspended
>
>+
>
>for those abusing reports: If you report N-number of approved flips during a ceremony then your reports get ignored

The identity will loose 30% out of mining rewards in case if at least 1 flip has been reported, if that happened again on the 2nd validation the identity gonna be penalized for 50% off the mining rewards and if that happened continuously on the 3rd time, identity gonna get suspended.

**Reply by ltraveler** [reply]
> by Teajay666:
>as far as false reports go, how about if you use ALL of your reports and a N-% (maybe 50%) of them are given to *approved flips, only then will your reports not count
>
>*approved also a N-%, for example 50%,... a flip gets 3 approves from a committee of 6
>
>so a real life example, someone uses all of their reports, say 6 reports and out of the 6 flips 4 are actually approved, then it's safe to say that this is a case of abusive usage of reports

I agree, if penalization gonna be related only to decriminalization in report vote rights (their reports wouldn't be counted at all) and to report rewards (they gonna loose them) in the current validation session: for example in case of 50% of false reports.
That is actually very smart idea, which could protect newbies because they are not gonna loose any other rewards apart from report rewards.

**Comment by 0JustinMiles0** [comment]
I think it's unnecessary to have a double penalty. We also want a maximum of identities to be mining, especially solo miners. Unfair reports happen and I think it would be too discouraging to solo miners to be penalized on their mining rewards. I think we should focus more on penalizing valid low quality flips. It seems to me that the reporting system has reached its limit in terms of what type of flips it penalizes.
